.github/workflows/rust.yml

```diff
@@ -85,6 +85,28 @@ jobs:
 
       - run: cargo check --locked --lib --all-features
 
+  minver:
```

Comment:
I guess we use `minver` in other repositories but the name triggers more of an MSRV notion for me than a `minimal-versions` notion. Maybe expand it? IMO brevity doesn't buy us that much here.